servicetalk-http-api/src/main/java/io/servicetalk/http/api/CharSequences.java

```diff
@@ -226,7 +225,7 @@ private static boolean isUpperCase(final char value) {
     }
 
     /**
-     * Tests if the string starts with the specified prefix.
+     * Tests if the string starts with the specified case-sensitive prefix.
```

Comment:
The content-coding names are case-insensitive. You do `contentEqualsIgnoreCase` first, then we should check the `startsWith` prefix case-insensitively.
Because the `startsWith` method is used only in one place, let's move it as a private method in `ContentCodings` class for now. Better to minimize the scope of public methods we have to maintain.

Reply:
Missed that, good catch